### PR TITLE
Implement Ember.Copyable without export an Ember.Object

### PR DESCRIPTION
**Pull request by mpirio**
Hello,

Here is a PR which "implements" `Ember.Copyable` without encapsulating moment object in an Ember.Object (see #148). All unit tests are OK.

### CODE REVIEW

**Comment by jasonmit** [comment]
I'll review tonight, thanks for spiking this :)

File: addon/index.js [hunk 1]
```diff
@@ -18,7 +75,17 @@ moment.prototype.compare = compare;
...
+}
+
+moment.prototype.copy = function copy(deep) {
+  return this.clone();
 }
 
-export default moment;
+const augmentedMoment = createAugmentedMomentFrom(moment);
+
+['utc', 'unix'].forEach((methodName) => {
```
Comment (jasonmit) [inline addon/index.js:87]: Is `tz` needed here since it also returns a moment object?

Comment (mpirio) [reply addon/index.js:87]: `tz`return a `moment.utc` instance (cf [moment-timezone.js#L488](https://github.com/moment/moment-timezone/blob/develop/moment-timezone.js#L488)) so no need here

File: addon/index.js [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+    }
+  });
+
+  return a;
+}
+
+function createAugmentedMomentFrom(fn) {
+  const augmentedMoment = function() {
+    const m = fn(...arguments);
+    const meta = Ember.meta(m);
```
Comment (jasonmit) [inline addon/index.js:47]: Private API I believe, makes me slightly worried...

Comment (mpirio) [reply addon/index.js:47]: Yes. I've not found how to do otherwise. The other solution is to use Ember.Object.

Comment (Techn1x) [reply addon/index.js:47]: Is this a show stopper?

Comment (jasonmit) [reply addon/index.js:47]: Yes.  I feel it will be painful to maintain.

Comment (mpirio) [reply addon/index.js:47]: so we have to use an `Ember.Object` to encapsulate Moment object... Perhaps could we use an [`Ember.ObjectProxy`](https://emberjs.com/api/ember/2.18/classes/ObjectProxy) to encapsultate it?

**Comment by jasonmit** [comment]
Closing as a result of #164

**Pull request closed without merging**